### DISCUSSION TITLE
extract frame information

### DISCUSSION THREAD

**Discussion by MichelMoser** [vdemichev/DiaNN#1027] [General]
Dear DIA-NN developers, 

Would it be possible to extract the frame number of the peak apex when reporting the XIC  for a given peptide (timsTOF data) or even all the reported peptides in the report.tsv-file?
 
Or is it possible to dig out the frame number already from the given output? 

Thank you very much, 
michel

**Comment by vdemichev** [comment]
Hi Michel,

Currently not possible. The reason is the difficulty of handling a situation when spectra from multiple frames are merged together by DIA-NN internally. 

Best,
Vadim

**Reply by MichelMoser** [reply]
Ok, thank you nevertheless! Great tool and it keeps getting better :)
